.github/workflows/build.yml

```diff
@@ -30,9 +30,6 @@ jobs:
           # Only Elixir 1.14 supports OTP 25
           - otp: "25.3"
             elixir: "1.14.4"
-          # We need to test Elixir 1.10 for backwards compatibility
-          - otp: "22.3.4.26"
-            elixir: "1.10.4"
```

Comment:
This should stay: we should bump here elixir to 1.11.x to make sure we are compatible with it.

Reply:
Replaced 1.10.4 with 1.11.4 (the latest 1.11 version).